src/structures/GuildAuditLogs.js

```diff
@@ -349,19 +349,21 @@ class GuildAuditLogsEntry {
             guild_id: guild.id,
           }));
     } else if (targetType === Targets.INVITE) {
-      if (guild.me.permissions.has('MANAGE_GUILD')) {
-        const change = this.changes.find(c => c.key === 'code');
-        this.target = guild.fetchInvites()
-          .then(invites => {
-            this.target = invites.find(i => i.code === (change.new || change.old));
-            return this.target;
-          });
-      } else {
-        this.target = this.changes.reduce((o, c) => {
-          o[c.key] = c.new || c.old;
-          return o;
-        }, {});
-      }
+      guild.members.fetch(guild.client.user.id).then(me => {
+        if (me.permissions.has('MANAGE_GUILD')) {
+          const change = this.changes.find(c => c.key === 'code');
+          this.target = guild.fetchInvites()
+            .then(invites => {
+              this.target = invites.find(i => i.code === (change.new || change.old));
+              return this.target;
+            });
+        } else {
+          this.target = this.changes.reduce((o, c) => {
```

Comment:
This statement should be followed by a `return this.target;` then to resolve with the correct value.